backend/src/modules/tournament-tables/core/application/command/AssignTournamentTablesCommandHandler.ts

```diff
@@ -15,9 +16,16 @@ export class AssignTournamentTablesCommandHandler implements CommandHandler<Assi
 
   async execute(command: AssignTournamentTables): Promise<CommandResult> {
     const tournamentId = command.tournamentId;
-    const tournamentTables = await this.repository.findByTournamentId(tournamentId);
+    const tournamentTables = await this.repository.findAllByTournamentId(tournamentId);
+    const tablesToAssign = command.tables.map((table) => {
+      return {
```

Comment:
Fajnie jakby na sam stół - TournamentTable też zrobić typ :) Ma to swoją nazwę i wiadomo co zawiera - numer i name.

Reply:
Taki refactor to może, jak już skończymy resztę zadań ;) A sam TournamentTable już istnieje, tyle, że zawiera jeszcze tournamentId

Reply:
OK :)